data/maps/objects/CeladonMansionRoofHouse.asm

```diff
@@ -1,3 +1,7 @@
+	object_const_def
+	const CELADONMANSION_ROOF_HOUSE_HIKER
+	const CELADONMANSION_ROOF_HOUSE_EEVEE_POKEBALL
```

Comment:
Same here (and various other maps), no spaces in the map name, and the constant name is just the sprite.
```suggestion
	const CELADONMANSIONROOFHOUSE_HIKER
	const CELADONMANSIONROOFHOUSE_POKE_BALL
```

Reply:
Would you be okay with: (They are essentially the same NPC.. one blocks the door to the vandalized house.. and the other is moved one space to the right.) Or would you prefer `GUARD1` and `GUARD2`?

```

	const CERULEANCITY_GUARD_MOVED
	const CERULEANCITY_GUARD_BLOCKING
```

Reply:
`GUARD1` and `GUARD2` is fine.

Reply:
Might have more of these I need to fix; so i'll leave this unresolved for now.

Reply:
Couple other questions in regards to this:

Do you want this changed to just `FIGHTINGDOJO_POKE_BALL1` and `FIGHTINGDOJO_POKE_BALL2` like the original suggestion? I think it is easier to read knowing which mon pokeball it is, personally.
```
	const FIGHTINGDOJO_HITMONLEE_POKE_BALL
	const FIGHTINGDOJO_HITMONCHAN_POKE_BALL
```

Also with the following, I currently call items by their item name not by their sprite; and pokemon are called by their pokemon name and not by `MONSTER` or whatever. It makes the code easier to read. 
```
	const CERULEANCAVE_B1F_MEWTWO
	const CERULEANCAVE_B1F_ULTRA_BALL
	const CERULEANCAVE_B1F_MAX_REVIVE
```

My opinion these are supposed to to be "OBJECT" constants; not sprite constants. So having them named closer to what they are in game, makes for easier to read code. The user can always just read the corresponding `object_event` if they want to know what sprite it is using.

Reply:
Couple other questions in regards to this:

Do you want this changed to just `FIGHTINGDOJO_POKE_BALL1` and `FIGHTINGDOJO_POKE_BALL2` like the original suggestion? I think it is easier to read knowing which mon pokeball it is, personally.
```
	const FIGHTINGDOJO_HITMONLEE_POKE_BALL
	const FIGHTINGDOJO_HITMONCHAN_POKE_BALL
```

Also with the following, I currently call items by their item name not by their sprite; and pokemon are called by their pokemon name and not by `MONSTER` or whatever. It makes the code easier to read. 
```
	const CERULEANCAVE_B1F_MEWTWO
	const CERULEANCAVE_B1F_ULTRA_BALL
	const CERULEANCAVE_B1F_MAX_REVIVE
```

My opinion these are supposed to to be "OBJECT" constants; not sprite constants. So having them named closer to what they are in game, makes for easier to read code. The user can always just read the corresponding `object_event` if they want to know what sprite it is using.